kubespawner/reflector.py

```diff
@@ -1,20 +1,19 @@
 import time
 import threading
 
-from traitlets.config import SingletonConfigurable
+from traitlets.config import LoggingConfigurable
 from traitlets import Any, Dict, Unicode
 from kubernetes import client, config, watch
 from tornado.ioloop import IOLoop
 
-class PodReflector(SingletonConfigurable):
+class NamespacedResourceReflector(LoggingConfigurable):
     """
-    Local up-to-date copy of a set of kubernetes pods
+    Local up-to-date copy of a set of kubernetes resources.
```

Comment:
Is this more accurate:

Base class for a local up-to-date copy of a type of kubernetes resource

Reply:
Yup! updated!